rules/completed-docs/index.html

```diff
@@ -1,9 +1,9 @@
 ---
 ruleName: completed-docs
-description: Enforces documentation for important items be filled out.
+description: Enforces documentation and JSDoc comments for important items be filled out.
```

Comment:
`documentation and JSDoc comments` seems redundant.
Maybe just `Enforces JSDoc for important ...`